datahub-web-react/src/app/shared/ManageAccount.tsx

```diff
@@ -65,8 +65,10 @@ export const ManageAccount = ({ urn: _urn, pictureLink: _pictureLink, name }: Pr
                     </MenuItem>
                 );
             })}
-            <MenuItem id="user-profile-menu-logout" danger key="logout" onClick={handleLogout} tabIndex={0}>
-                Log out
+            <MenuItem danger key="logout" tabIndex={0}>
```

Comment:
Have you validated that this still works for both OIDC and non-oidc (jaas) authentication modes?

Reply:
Yes, we validated in our env that it works for both OIDC (keycloak) and non-oidc authentication modes